<launch>
  <arg name="name" default="mask_rcnn"/>
  <arg name="cfg" default="e2e_mask_rcnn_R-101-FPN_2x.yaml"/>
  <arg name="wts" default="model_final.pkl"/>
  <arg name="train_wts" default="R-101.pkl"/>
  <arg name="confidence" default="0.9"/>
  <arg name="sub_img_topic" default="/scene/left/image_rect_color"/>
  <arg name="frame_rate" default="30"/>
  <arg name="gpu_id" default="0"/>
  <arg name="mask_on" default="True"/>
  <node pkg="macaca_mask_rcnn" type="infer.py" name="$(arg name)" output="screen">
    <param name="cfg" type="str" value="$(find macaca_mask_rcnn)/config/$(arg cfg)"/>
    <param name="wts" type="str" value="$(find macaca_mask_rcnn)/config/$(arg wts)"/>
    <param name="train_wts" type="str" value="$(find macaca_mask_rcnn)/config/$(arg train_wts)"/>
    <param name="confidence" type="double" value="$(arg confidence)"/>
    <param name="sub_img_topic" type="str" value="$(arg sub_img_topic)"/>
    <param name="frame_rate" type="int" value="$(arg frame_rate)"/>
    <param name="gpu_id" type="int" value="$(arg gpu_id)"/>
    <param name="mask_on" type="bool" value="$(arg mask_on)"/>
  </node>
</launch>
